src/main/java/org/opentripplanner/routing/api/request/RoutingService.java

```diff
@@ -1,4 +1,4 @@
-package org.opentripplanner.routing;
+package org.opentripplanner.routing.api.request;
```

Comment:
The RoutingService is the main entry point of the `org.opentripplanner.routing` package and should be in the root.